src/plugins/intel_cpu/tests/functional/behavior/export_import.cpp

```diff
@@ -62,4 +62,47 @@ TEST(ExportImportTest, ExportOptimalNumStreams) {
         EXPECT_EQ(nstreams_latency_original, nstreams_latency_imported);
     }
 }
+
+class ExportOptimalNumStreamsTest : public ::testing::TestWithParam<std::string> {};
+
+TEST_P(ExportOptimalNumStreamsTest, OptimalNumStreamsThroughput) {
+    auto original_model = MakeMatMulModel();
```

Comment:
There are a lot of duplicated code for this test, why not use `INSTANTIATE_TEST_SUITE_P`?

Reply:
Ok, removed.